src/Common/src/System/Drawing/KnownColorTable.cs

```diff
@@ -3,550 +3,292 @@
+            0xFFFFFFE0,     // LightYellow
+            0xFF00FF00,     // Lime
+            0xFF32CD32,     // LimeGreen
+            0xFFFAF0E6,     // Linen
+            0xFFFF00FF,     // Magenta
+            0xFF800000,     // Maroon
+            0xFF66CDAA,     // MediumAquamarine
+            0xFF0000CD,     // MediumBlue
+            0xFFBA55D3,     // MediumOrchid
+            0xFF9370DB,     // MediumPurple
+            0xFF3CB371,     // MediumSeaGreen
+            0xFF7B68EE,     // MediumSlateBlue
+            0xFF00FA9A,     // MediumSpringGreen
+            0xFF48D1CC,     // MediumTurquoise
+            0xFFC71585,     // MediumVioletRed
+            0xFF191970,     // MidnightBlue
+            0xFFF5FFFA,     // MintCream
+            0xFFFFE4E1,     // MistyRose
+            0xFFFFE4B5,     // Moccasin
+            0xFFFFDEAD,     // NavajoWhite
+            0xFF000080,     // Navy
+            0xFFFDF5E6,     // OldLace
+            0xFF808000,     // Olive
+            0xFF6B8E23,     // OliveDrab
+            0xFFFFA500,     // Orange
+            0xFFFF4500,     // OrangeRed
+            0xFFDA70D6,     // Orchid
+            0xFFEEE8AA,     // PaleGoldenrod
+            0xFF98FB98,     // PaleGreen
+            0xFFAFEEEE,     // PaleTurquoise
+            0xFFDB7093,     // PaleVioletRed
+            0xFFFFEFD5,     // PapayaWhip
+            0xFFFFDAB9,     // PeachPuff
+            0xFFCD853F,     // Peru
+            0xFFFFC0CB,     // Pink
+            0xFFDDA0DD,     // Plum
+            0xFFB0E0E6,     // PowderBlue
+            0xFF800080,     // Purple
+            0xFFFF0000,     // Red
+            0xFFBC8F8F,     // RosyBrown
+            0xFF4169E1,     // RoyalBlue
+            0xFF8B4513,     // SaddleBrown
+            0xFFFA8072,     // Salmon
+            0xFFF4A460,     // SandyBrown
+            0xFF2E8B57,     // SeaGreen
+            0xFFFFF5EE,     // SeaShell
+            0xFFA0522D,     // Sienna
+            0xFFC0C0C0,     // Silver
+            0xFF87CEEB,     // SkyBlue
+            0xFF6A5ACD,     // SlateBlue
+            0xFF708090,     // SlateGray
+            0xFFFFFAFA,     // Snow
+            0xFF00FF7F,     // SpringGreen
+            0xFF4682B4,     // SteelBlue
+            0xFFD2B48C,     // Tan
+            0xFF008080,     // Teal
+            0xFFD8BFD8,     // Thistle
+            0xFFFF6347,     // Tomato
+            0xFF40E0D0,     // Turquoise
+            0xFFEE82EE,     // Violet
+            0xFFF5DEB3,     // Wheat
+            0xFFFFFFFF,     // White
+            0xFFF5F5F5,     // WhiteSmoke
+            0xFFFFFF00,     // Yellow
+            0xFF9ACD32,     // YellowGreen
+        };
 
-        public static Color ArgbToKnownColor(int targetARGB)
+        internal static Color ArgbToKnownColor(uint argb)
         {
-            EnsureColorTable();
-            for (int index = 0; index < s_colorTable.Length; ++index)
+            // Should be fully opaque
```

Comment:
It took me a minute to realize that this comment is alluding to why we're starting the iteration at index 1 instead of 0.  It'd be good to clarify that further, that the 0th index is for Transparent, and hence we can skip it.